applications/openshift/networking/default_ingress_ca_replaced/tests/ocp4/e2e-remediation.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
 
-# we are using an existing default secret name for testing, so it won't cause any subsequent failures on testing routes. 
+# Generate a new certificate. The details of the certificate don't really
+# matter since we simply need it for the ConfigMap so that the cluster
+# operators come back up after the trustedCA change.
+openssl req -x509 -newkey rsa:4096 -keyout /tmp/key.pem -out /tmp/cert.pem -sha256 -days 365 -nodes -subj "/C=XX/ST=StateName/L=CityName/O=CompanyName/OU=CompanySectionName/CN=CommonNameOrHostname"
+oc create configmap trusted-ca-bundle --from-file=ca-bundle.crt=/tmp/cert.pem -n openshift-config
```

Comment:
We could use these commands, right? 

https://docs.openshift.com/container-platform/4.15/security/certificates/replacing-default-ingress-certificate.html